frontend/components/messenger/message_list.jsx

```diff
@@ -22,21 +21,21 @@ class MessageList extends React.Component{
 
   componentDidUpdate(){
     const { selectedId, receiveMessage } = this.props;
-    document.getElementById('overflow').scrollIntoView({behavior: "smooth"});
+    document.getElementById('overflow').scrollIntoView({ behavior: "smooth" });
   }
 
   render(){
-    const { messages } = this.props;
+    const { messages, selectedId, users, currentUserId } = this.props;
     const relevantMessages = Object.values(messages).filter( message => {
-      return message.conversationable_id === this.props.selectedId;
+      return message.conversationable_id === selectedId;
     });
     const messageEls = relevantMessages.map( message => {
-      const user = this.props.users[message.user_id];
+      const user = users[message.user_id];
       return <MessageListItem
                 key={message.id}
```

Comment:
same thing in this return here. open parens and on next line start you component. the reason for this convention is that all of our return values from `render()` functions are multi-line returns with parens. it keeps it easy to read if its the same across all returns.